os/FlatLibs.mk

```diff
@@ -175,6 +175,10 @@ ifeq ($(CONFIG_AMEBAD_WIFI),y)
 TINYARALIBS += $(LIBRARIES_DIR)$(DELIM)lib_wlan$(LIBEXT)
 endif
 
+ifeq ($(CONFIG_AMEBAD_BLE),y)
```

Comment:
Let's add this in amebad's Makefile, not in common.
Please refer below.
https://github.com/Samsung/TizenRT/blob/master/os/arch/arm/src/amebad/Make.defs#L169